apps/dashboard/app/views/projects/edit.html.erb

```diff
@@ -1,7 +1,20 @@
 <div class='page-header text-center'>
-  <h1>Edit Project</h1>
+  <h1>
+    Editing: <%= @project.name %>
+  </h1>
 </div>
 
 <br>
 
-<%= render 'form' %>
+<%= bootstrap_form_for(@project, url: project_path, method: :patch) do |form| %>
+  <%= render "form_manifest", form: form %>
```

Comment:
I don't think we want manifest in the name here.

```suggestion
  <%= render "project_edit_form", form: form %>
```

Reply:
I missed this on the last commit but will make the adjustment.